Annotations/AddAnnotation/packages.config

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Syncfusion.Compression.Base" version="20.3.0.47" targetFramework="net472" />
-  <package id="Syncfusion.Licensing" version="20.3.0.47" targetFramework="net472" />
-  <package id="Syncfusion.Pdf.Wpf" version="20.3.0.47" targetFramework="net472" />
-  <package id="Syncfusion.PdfViewer.WPF" version="20.3.0.47" targetFramework="net472" />
-  <package id="Syncfusion.Shared.WPF" version="20.3.0.47" targetFramework="net472" />
+  <package id="Syncfusion.Compression.Base" version="*" targetFramework="net472" />
+  <package id="Syncfusion.Licensing" version="*" targetFramework="net472" />
+  <package id="Syncfusion.Pdf.Wpf" version="*" targetFramework="net472" />
+  <package id="Syncfusion.PdfViewer.WPF" version="*" targetFramework="net472" />
```

Comment:
Where is the reference for the pdfToImageConverter

Reply:
Syncfusion.PdfViewer.Wpf single reference sample is working so removed another reference